cmd/qat_plugin/qat_plugin.go

```diff
@@ -41,12 +41,13 @@ func main() {
 
 	dpdkDriver := flag.String("dpdk-driver", "vfio-pci", "DPDK Device driver for configuring the QAT device")
 	kernelVfDrivers := flag.String("kernel-vf-drivers", "c6xxvf,4xxxvf", "Comma separated VF Device Driver of the QuickAssist Devices in the system. Devices supported: DH895xCC, C62x, C3xxx, C4xxx, 4xxx, and D15xx")
+	preferredAllocationPolicy := flag.String("allocation-policy", "", "Modes of allocating QAT devices: balanced and packed")
+
 	maxNumDevices := flag.Int("max-num-devices", 32, "maximum number of QAT devices to be provided to the QuickAssist device plugin")
-	flag.Parse()
```

Comment:
parse is needed here because `mode` needs to be checked

Reply:
Ah, It was a mistake. I moved accidently the parse to another function in another file. Thank you for your review.!